tensorflow/contrib/lite/examples/label_image/label_image.cc

```diff
@@ -148,14 +148,22 @@ void RunInference(Settings* s) {
   int wanted_width = dims->data[2];
   int wanted_channels = dims->data[3];
 
-  if (s->input_floating) {
-    downsize<float>(interpreter->typed_tensor<float>(input), in, image_height,
-                    image_width, image_channels, wanted_height, wanted_width,
-                    wanted_channels, s);
-  } else {
-    downsize<uint8_t>(interpreter->typed_tensor<uint8_t>(input), in,
-                      image_height, image_width, image_channels, wanted_height,
-                      wanted_width, wanted_channels, s);
+  switch (interpreter->tensor(input)->type) {
+    case kTfLiteFloat32:
+      s->input_floating = true;
+      downsize<float>(interpreter->typed_tensor<float>(input), in, image_height,
+                      image_width, image_channels, wanted_height, wanted_width,
+                      wanted_channels, s);
```

Comment:
please make sure the lines are less than 80-chars long and properly wrapped.

Reply:
This is exactly 80-chars and conformed to the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html). That's why it didn't catch by clang-format.